HTML/EN/plugins/Qobuz/settings/basic.html

```diff
@@ -51,7 +51,7 @@
 			<select name="pref_workPlaylistPosition" id="workPlaylistPosition">
 				<option [% prefs.workPlaylistPosition == "before" ? "selected" : "" %] value="before" >[% "PLUGIN_QOBUZ_WORK_PLAYLIST_POSITION_BEFORE" | string %]</option>
 				<option [% prefs.workPlaylistPosition == "after"  ? "selected" : "" %] value="after" >[% "PLUGIN_QOBUZ_WORK_PLAYLIST_POSITION_AFTER" | string %]</option>
-				<option [% prefs.workPlaylistPosition == "hidden" ? "selected" : "" %] value="hidden" >[% "PLUGIN_QOBUZ_WORK_PLAYLIST_POSITION_HIDDEN" | string %]</option>
+				<option [% prefs.workPlaylistPosition == "integrated" ? "selected" : "" %] value="integrated" >[% "PLUGIN_QOBUZ_WORK_PLAYLIST_POSITION_INTEGRATED" | string %]</option>
```

Comment:
This would be a breaking change to previous revisions. Is there any migration possible?

Reply:
I did test this: if the user has the "hidden" option selected, it just behaves the same as it did before, until they go to preferences and re-save. 

But perhaps it would be better if we automatically updated "hidden" to "integrated"?

Reply:
If it behaves as before, then that's fine.